index.html

```diff
@@ -0,0 +1,514 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/index.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,500;0,600;1,500&display=swap"
+      rel="stylesheet"
+    />
+    <title>BEM practice | KMarketer</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <h1 class="title title--letter-spacing nav__title--left-spacing">
+          KMarketer
+        </h1>
+        <ul class="menu menu--flex">
+          <li class="item">Home</li>
+          <li class="item">Products</li>
+          <li class="item">Pricing</li>
+          <li class="item">Contact</li>
+          <li class="item item--flex">
+            <span>
```

Comment:
Contains `SPAN` elements with no class. These are unnecessary. As, you are using flex you can achieve what those `SPAN` doing with flex.